test/test_nested_tensor_functional.py

```diff
@@ -259,6 +259,13 @@ def test_nn_functional_interpolate(self):
                 nt_res = torch.nn.functional.interpolate(nt, size, mode='bilinear', align_corners=True)
                 self.assertEqual(nestedtensor.nested_tensor(tensor_res), nt_res)
 
+        # special NT case - list of sizes
+        size = ((100, 100), (200, 250))
+        for nt in [nestedtensor.nested_tensor(inputs), nestedtensor.as_nested_tensor(inputs)]:
```

Comment:
You could also test whether broadcasting works, i.e. only passing ```size = ((100, 100),)``` and what happens if you pass an empty list.